src/pudl/metadata/sources.py

```diff
@@ -216,12 +243,17 @@
     "ferc1": {
         "title": "FERC Form 1",
         "path": "https://www.ferc.gov/industries-data/electric/general-information/electric-industry-forms/form-1-electric-utility-annual",
-        "description": (
-            "The Federal Energy Regulatory Commission (FERC) Form 1 is a "
-            "comprehensive financial and operating report submitted annually for "
-            "electric rate regulation, market oversight analysis, and financial audits "
-            "by Major electric utilities, licensees and others."
-        ),
+        "description": {
+            "source_description": (
+                "The Federal Energy Regulatory Commission (FERC) Form 1 is a "
+                "comprehensive financial and operating report submitted annually for "
+                "electric rate regulation, market oversight analysis, and financial audits "
+                "by Major electric utilities, licensees and others."
+            ),
+            "respondents": "Major electric utilities and licenses.",
+            "records_liberated": "~13.2 million (116 raw tables), ~307,000 (7 clean tables)",
+            "source_format": "XBRL (.XBRL)",
```

Comment:
Should be a mix of XBRL and Visual FoxPro DBF.

Reply:
Will it ever be fully transitioned to XBRL or always a mix of XBRL and Visual FoxPro?